app/src/main/java/com/example/voyagerx/ui/fragments/userprofileac/FavoritesAdapter.kt

```diff
@@ -4,40 +4,42 @@ import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import com.example.voyagerx.databinding.FavoritesOverviewCardBinding
+import com.example.voyagerx.databinding.LandingPageOverviewCardBinding
 import com.example.voyagerx.helpers.LaunchClickListener
 import com.example.voyagerx.repository.model.User
+import com.example.voyagerx.ui.fragments.landing.list.LaunchOverviewViewHolder
 import com.example.voyagerx.ui.fragments.userprofileac.FavoritesAdapter.*
 
 
 class FavoritesAdapter(
     private val user: User?,
     private val listener: LaunchClickListener = LaunchClickListener { }
 ) :
-    RecyclerView.Adapter<ViewHolder>() {
-
-
-    class ViewHolder(val binding: FavoritesOverviewCardBinding) : RecyclerView.ViewHolder(binding.root)
+    RecyclerView.Adapter<LaunchOverviewViewHolder>() {
```

Comment:
This makes a lot more sense with the reusage of Chris' cards. Really great catch!